### DISCUSSION TITLE
How to close the discussion on github?

### DISCUSSION THREAD

**Discussion by JedediahXu** [community/community#28745] [Enterprise] [answered]
How to close the discussion on github?
Can the discussion be deleted please?
I posted a discussion thread the other day, and I want to delete or close it! What should I do please?

**Comment by GabyCT** [comment, marked as answer]
Yes a discussion can be deleted, you can see it here https://docs.github.com/en/enterprise-cloud@latest/discussions/managing-discussions-for-your-community/managing-discussions#deleting-a-discussion

**Comment by jfullstackdev** [comment]
you mean this ? you can see the options bottom right after participants

![image](https://user-images.githubusercontent.com/47092464/184069665-a26d45fc-90e5-4cfb-b35b-88fc3dd2746d.png)

if you are the owner of that Discussion, you can delete or lock it, you may be referring to closing it as locking it
